docs/en/metadata_oidc_rp.rst

```diff
@@ -95,6 +95,9 @@ The RP Metadata of type **"openid_relying_party"** MUST contain at least the fol
   * - **client_registration_types**
     - See `OIDC-FED`_ Section 4.1. The required value is **automatic**. 
     - |spid-icon| |cieid-icon|
+  * - **response_types**
```

Comment:
Non era stato inserito perchè opzionale e il suo valore di default è proprio code.
response_types
OPTIONAL. JSON array containing a list of the OAuth 2.0 response_type values that the Client is declaring that it will restrict itself to using. If omitted, the default is that the Client will use only the code Response Type.

Inoltre nel metadata OP è presente il valore response_type_supported che DEVE essere valorizzato con code. 

Dal mio punto di vista aggiungerlo è superfluo

Reply:
effettivamente un RP può partecipare a federazioni multiple e malauguratamente supportare altri flussi oltre a code

@damikael leggo da LLGG SPID OIDC che "Deve" (non uppercased, quindi normativo-debole)

@fmarino-ipzs vogliamo specificare che DEVE solo per SPID qui oppure va bene DEVE anche per CIE?

Reply:
@peppelinux , se effettivamente un RP può partecipare ad altre federazioni che supportano altri flussi oltre a code, forse sarebbe meglio riportare "It MUST contain the value **code**." (senza "only"). Che ne pensi? 
Per quanto riguarda la CIE per me va bene anche il MUST, tanto, come già detto, quello che poi conta è il response_type_supported nel metadata OP.

Reply:
eccolo afca7d53462c76b30cce90390b01320f9220209f